src/Pages/UserPage/Components/Likes.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+import { useState, useEffect } from "react";
+import FooterIcon from "./FooterIcon";
```

Comment:
```suggestion
import React, { useState, useEffect } from "react";
import { Link } from "react-router-dom";
import styled from "styled-components";
import FooterIcon from "./FooterIcon";
```
import 순서 맞춰주시고 react 두번 불린거 합쳐주세요

Reply:
수정했습니다 😂